chromium_src/content/child/runtime_features.cc

```diff
@@ -1,12 +1,24 @@
-#define SetRuntimeFeaturesDefaultsAndUpdateFromArgs SetRuntimeFeaturesDefaultsAndUpdateFromArgs_ChromiumImpl
+/* Copyright (c) 2018 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define SetRuntimeFeaturesDefaultsAndUpdateFromArgs \
+  SetRuntimeFeaturesDefaultsAndUpdateFromArgs_ChromiumImpl
+
 #include "../../../../content/child/runtime_features.cc"
+
 #undef SetRuntimeFeaturesDefaultsAndUpdateFromArgs
 
 namespace content {
 
 void SetRuntimeFeaturesDefaultsAndUpdateFromArgs(
     const base::CommandLine& command_line) {
   SetRuntimeFeaturesDefaultsAndUpdateFromArgs_ChromiumImpl(command_line);
+
+  // Brave will use its own mechanism for farbling the actual list of plugins
+  // returned by navigator.plugins depending on the selected BraveFarblingLevel.
+  WebRuntimeFeatures::EnableNavigatorPluginsFixed(false);
```

Comment:
this one is controlled via a feature, maybe we could use existing machinery for overriding features instead of overrides?

Reply:
@mariospr

Reply:
Haven't looked up the feature, but we also disable some blink features in `renderer/brave_content_renderer_client.cc`

Reply:
Switched to disabling via the feature in 9368f12fbb89b06e6d763c05be24f880ab3b29cb

Reply:
I apologize because I missed this mail (and others) last week and couldn't reply on time. In any case I agree with the resolution in 9368f12fbb89b06e6d763c05be24f880ab3b29cb, my only doubt is whether it makes sense to keep chromium_src/content/child/runtime_features.cc around at all, now that `SetRuntimeFeaturesDefaultsAndUpdateFromArgs()` is basically calling `SetRuntimeFeaturesDefaultsAndUpdateFromArgs_ChromiumImpl()`.

WDYT?

Reply:
I think it might have been left as a placeholder, but ok with me either way.

Reply:
Yes, I had the same feeling (re: placeholder) when I used it for my patch, and I'm not 100% clear on whether to remove it tbh.